src/main/java/service/imp/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.imp;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import java.util.Map;
+import java.util.Set;
+import service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
The class `ReportGeneratorImpl` should not begin with an empty line according to the checklist.